bridge/svix-bridge/src/config/mod.rs

```diff
@@ -238,5 +253,61 @@ impl WebhookReceiverConfig {
     }
 }
 
+#[derive(Clone, Deserialize)]
+#[serde(tag = "type", rename_all = "kebab-case")]
+pub enum PollerInputOpts {
+    SvixEvents {
+        app_id: String,
+        subscription_id: String,
+        svix_token: String,
+        #[serde(default)]
+        svix_options: Option<SvixOptions>,
+    },
+}
+
+impl PollerInputOpts {
+    pub fn svix_client(&self) -> Option<Svix> {
+        match self {
+            PollerInputOpts::SvixEvents {
+                svix_token,
+                svix_options,
+                ..
+            } => Some(Svix::new(
+                svix_token.clone(),
+                svix_options.clone().map(Into::into),
+            )),
+        }
+    }
```

Comment:
Looks like this always returns `Some`, why wrap the `Svix` value in an `Option` at all?